.github/workflows/create-draft-release.yml

```diff
@@ -0,0 +1,60 @@
+name: "Create Draft Release"
+
+env:
+  BUILDDIR: "build"
+  DEVELOPER_DIR: "/Applications/Xcode_12.5.1.app/Contents/Developer"
+
+on:
+  workflow_dispatch
+
+concurrency: 
+  group: publish-release-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    name: "Publish binaries for release"
+    runs-on: macos-11
+
+    steps:
+      - name: "Checkout sources"
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+          fetch-depth: 0
+
+      - name: "Build release distribution"
+        run: make release
+        env:
+          GITHUB_ACTOR: ${{ github.actor }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: "Extract version information from project"
+        run: |
+          MARKETING_VERSION=$(xcrun xcodebuild -showBuildSettings -project Sparkle.xcodeproj | grep MARKETING_VERSION | sed -E 's/.+=[[:space:]]+(.*)$/\1/')
```

Comment:
I wish xcodebuild just had a command to properly extract individual build settings rather than regex-hacking it.

This could possibly break if MARKETING_VERSION = happens to be inserted in some other place, like `GCC_PREPROCESSOR_DEFINITIONS` for example, which is a feasible thing to happen in the future. So maybe you could make it more resilient at least so that the line (ignoring whitespace) needs to start with MARKETING_VERSION, and it would be a bad error if more than one match was found.

Otherwise at the very least we may need to add a comment in the CommonConfig file being careful about using/exporting MARKETING_VERSION elsewhere.

Reply:
Yeah, I wish this were true, too. 

I can't think of a great safety check for other variables containing MARKETING_VERSION here, so I'll add a comment to the Xcode config so that we can keep moving.

Reply:
I've added the comment in 18b5da5db03e17bb20144c2fd6e85ee6f55d5ada.